<?xml version="1.0"?>
<launch>
<include file="$(find cpc_reference_publisher)/launch/ref_pub.launch"/>
<node pkg="cpc_motion_planning" type="cpc_motion_planning_node" name="cpc_motion_planning_node" output="screen">
<remap from="/raw_odom" to ="/raw_odom_delayed"/>
<remap from="/slam_odom" to ="/slam_odom_delayed"/>
</node>
</launch>
